istqb-ctel-tm-mtt-complete-guide.md

```
@@ -0,0 +1,1687 @@
+# 👥 ISTQB® Expert Level Test Management – Managing the Test Team (CTEL-TM-MTT)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)